gateway/proxy.go

```diff
@@ -7,13 +7,25 @@ import (
 	"net/url"
 )
 
+const (
+	DISTRIBUTOR = "distributor"
+)
+
+var defaultTimeoutValues map[string]Timeouts = map[string]Timeouts{
+	DISTRIBUTOR: {
+		ReadTimeout:  5,
+		WriteTimeout: 5,
+		IdleTimeout:  5,
+	},
+}
```

Comment:
I gave default values as 5's but it was random. What should be the correct timeout values for the distributor?

Reply:
Only testing can tell. It's fine for now